### PR TITLE
interim helm update

### PR DESCRIPTION
**Pull request by morgo**
I have managed to get helm 3.0 to work, with some small changes I will add in a PR soon.

Kubernetes 1.16 looks like it might be a little bit of work, so I will add a note for now.

### CODE REVIEW

**Comment by derekperkins** [comment]
I think this might be a little misleading because you can't install the etcd-operator with helm 3 yet. I spent 30 minutes at the Vitess station trying to get things working with a guy from Digital Ocean before that was the final stopping point.
https://github.com/helm/charts/issues/19008

**Comment by morgo** [comment]
> I think this might be a little misleading because you can't install the etcd-operator with helm 3 yet. I spent 30 minutes at the Vitess station trying to get things working with a guy from Digital Ocean before that was the final stopping point.
> [helm/charts#19008](https://github.com/helm/charts/issues/19008)

Hmm.. I thought I got it to work with helm 3. I'll have to take another look.

**Comment by derekperkins** [comment]
IIUC, the installation works, but it uses a helm 2 hook to create the CRDs themselves. If you already had the definitions themselves, the rest would have worked.

**Comment by derekperkins** [comment]
What would be ideal for the getting started guide would be to use the Kubernetes apii server as the topo server. @carsonoid started on an implementation but didn't finish it. I think one of the biggest questions is how much it might bloat the binary to include the k8s client.
https://github.com/carsonoid/vitess/tree/kubernetestopo

**Comment by carsonoid** [comment]
I was just thinking about the K8s topo server implementation today. I would love to get a chance to complete it.

**Comment by morgo** [comment]
@derekperkins PTAL again

File: content/en/docs/get-started/kubernetes.md [hunk 1]
```diff
@@ -5,39 +5,55 @@ featured: true
...
+example/rbac/create_role.sh
+kubectl create -f example/deployment.yaml
+```
 
-* Install the MySQL client. On Ubuntu: `apt-get install mysql-client`
-* Install vtctlclient
+4. Install the MySQL client locally. For example, on Ubuntu: `apt-get install mysql-client`
+5. Install vtctlclient locally:
     * Install go 1.12+
     * `go get vitess.io/vitess/go/cmd/vtctlclient`
```
Comment (derekperkins) [inline content/en/docs/get-started/kubernetes.md:45]: Does this start to break with go modules? For the future, we should provide a download link to the latest versioned release.

**Comment by derekperkins** [comment]
LGTM

**Review by derekperkins — APPROVED** [review]

File: content/en/docs/get-started/kubernetes.md (hunk 1 above)
Comment (morgo) [reply content/en/docs/get-started/kubernetes.md:45]: Still worked for me, but maybe not the smartest way to do things. I will change it once we publish the new (smaller) package format.

**Pull request merged**